assets/src/components/AssignmentTable.js

```diff
@@ -195,8 +202,7 @@ function AssignmentTable (props) {
                               id='standard-number'
                               value={a.goalGrade}
                               label={
-                                !courseGoalGradeSet
-                                  ? 'Set course goal'
+                                !courseGoalGradeSet ? 'Set a goal'
```

Comment:
It is suggested internally to use `Set a goal` instead of for `Set course goal` assignments since the word `course` doesn't make sense there